cognite/neat/graph/loaders/core/rdf_to_assets.py

```diff
@@ -174,6 +173,9 @@ def _define_asset_class_mapping(transformation_rules: Rules) -> dict[str, dict[s
                         asset_class_mapping[class_][resource_type_property] += [property_.property_name]
 
                     if property_.property_name not in asset_class_mapping[class_]["metadata"]:
+                        # Todo; Why Nikola?  This adds for example name property to metadata? Isn't that
+                        #   controlled by the resource_type_property? If you would like this behavior you
+                        #   would set resource_type_property to ["metadata", "name"]?
```

Comment:
This threw me for a loop. Do you know why this is done?

Reply:
This was due to early requests by our customers that by default all properties end up in metadata. 
This was a quick fix (at that time) to make sure that all properties end up as keys in `Asset.metadata`

However, I think we should enforce explicit configuration, and start removing implicit behavior of neat.

Reply:
feel free to remove this implicit behavior, make sure that we state it in changelog